src/main/java/fizzbuzz/HitCondition.java

```diff
@@ -0,0 +1,9 @@
+package fizzbuzz;
+
+interface HitCondition {
```

Comment:
interface does not need more segregation I guess - since all implementations can support all methods in a meaningful way